nym-wallet/src/components/Bonding/BondedMixnode.tsx

```diff
@@ -63,6 +66,7 @@ export const BondedMixnode = ({
   network?: Network;
   onActionSelect: (action: TBondedMixnodeActions) => void;
 }) => {
+  const [nextEpoch, setNextEpoch] = useState<string | undefined | Error>();
```

Comment:
I don't think you need to specify undefined as a type if the state begins as undefined.

Reply:
@fmtabbara you are right, changed!